src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -297,6 +297,49 @@ protected function getArrayableItems(array $values)
         return $values;
     }
 
+    /**
+     * Determine if the given attribute exists.
+     *
+     * @param  string  $key
+     * @return bool
+     */
+    public function hasAttribute($key)
+    {
+        if (! $key) {
```

Comment:
does this mean the key can be null? If so, the phpdoc should be `string|null`. If not, this line can be removed.

Reply:
I've copied it from `getAttribute` - unsure why it even exist.

Should i remove the line or update the docbloc?